rust_icu_uloc/src/lib.rs

```diff
@@ -267,6 +332,91 @@ impl ULoc {
         let asciiz = self.as_c_str();
         buffered_string_char_star(uloc_method, asciiz.as_ptr())
     }
+
+    /// Call a `uloc` method that takes this locale's ID, panics on any errors, and returns
+    /// `Some(result)` if the result is non-empty, or `None` otherwise.
+    fn call_buffered_string_method_to_option(
```

Comment:
Isn't this just a [`Result::ok()`](https://doc.rust-lang.org/std/result/enum.Result.html#method.ok)?

Reply:
This is checking the length of the actual `String`. I changed a comment and added a type annotation to clarify.